samples/Compute/deploy-dsc-extension/README.md

```diff
@@ -0,0 +1,24 @@
+## Deploy DSC Extension to Azure VM and Arc connected machines
+
+Deploys the DSC extension to and assigns configuration artifact from url location.
+
+## Try on Portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FAutomation%2Fdeploy-dsc-extension%2Fazurepolicy.json)
```

Comment:
All URLs should use samples/Compute/ in this file

Reply:
fixed